osdu/README.md

```diff
@@ -0,0 +1,17 @@
+# OSDU - Open Subsurface Data Universe Ontology
+This is an open source ontology for the subsurface energy data based on [schema files and standards](https://community.opengroup.org/osdu/platform/data-flow/data-loading/open-test-data/-/tree/master/rc--3.0.0/3-schema) specified by the [Open Subsurface Data Universe](https://osduforum.org/) community.
+
+# License
+[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+The OSDU Ontology is licensed under the Apache License 2.0 - see [License](./License).
+
+
+## Homepage
+[OSDU Ontology](https://github.com/Accenture/OSDU-Ontology/)
+
+## Vocabulary Usage
+
+    @prefix osdu: <https://w3id.org/osdu#>
```

Comment:
https://github.com/Accenture/OSDU-Ontology/blob/main/ttl/OSDU.ttl is using https://w3id.org/OSDU# vs https://w3id.org/osdu# used here and the directory itself..  This service is case sensitive.